src/DependencyInjection/Configuration.php

```diff
@@ -118,6 +118,12 @@ private function addContextsSection(ArrayNodeDefinition $node): void
                                         ->scalarNode('format')->defaultValue('jpg')->end()
                                         ->scalarNode('constraint')->defaultValue(true)->end()
                                         ->scalarNode('resizer')->defaultNull()->end()
+                                        ->arrayNode('resizer_options')
+                                            ->info('options directly pass to selected resizer. e.g. {use_crop: true, crop_gravity: center}')
```

Comment:
Is this better grammar:
```suggestion
                                            ->info('options directly passed to selected resizer. e.g. {use_crop: true, crop_gravity: center}')
```
?